### PR TITLE
[New Rule] Potential Local NTLM Relay via HTTP

### PR DESCRIPTION
**Pull request by Samirbous**
Detect attempt to elevate privileges via coercing a privileged service to connect to a local rogue HTTP endpoint, leading to NTLM relay, example of logs while testing https://github.com/med0x2e/NTLMRelay2Self (step 5):

image

### CODE REVIEW

**Review by imays11 — APPROVED** [review]
-Looks Good

File: rules/windows/credential_access_relay_ntlm_auth_via_http_spoolss.toml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+rule_id = "4682fd2c-cfae-47ed-a543-9bed37657aa6"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.name : "rundll32.exe" and
+  /* relay to HTTP via spoolss or srvsvc pipes */
+  process.command_line : ("*davclnt.dll*http*spoolss*", "*davclnt.dll*http*pipe*srvsvc", "*davclnt.dll*http*/print/PIPE/*")
```
Comment (Mikaayenson) [inline rules/windows/credential_access_relay_ntlm_auth_via_http_spoolss.toml:32]: Is `"*davclnt.dll*http*pipe*srvsvc"` needed? I didn't see anything in your screenshot that matched on this.

**Review by DefSecSentinel — APPROVED** [review]
LGTM

File: rules/windows/credential_access_relay_ntlm_auth_via_http_spoolss.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/credential_access_relay_ntlm_auth_via_http_spoolss.toml]: image

search was KQL (case sens, that's why it didn't show up)

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Pull request merged**